docs/self_hosting/configuration/external_clickhouse.mdx

```diff
@@ -33,7 +33,7 @@ Additionally, sensitive information can be configured to be not stored in Clickh
 
 - A provisioned ClickHouse instance that your LangSmith application will have network access to (see above for options).
 - A user with admin access to the ClickHouse database. This user will be used to create the necessary tables, indexes, and views.
-- We only support standalone ClickHouse (not clustered or replicated) or ClickHouse Cloud.
+- We support both standalone ClickHouse and externally managed clustered deployments. For clustered deployments, ensure all nodes are running the same version. Note that clustered setups are not supported with LangSmith-managed ClickHouse installations.
```

Comment:
Same here as below on "not supported". 

For LangSmith-managed ClickHouse installations you can say:  Note that Langsmith-managed Clickhouse installations are automatically clustered as they are using SharedMerge, which offers separation of compute and storage.